src/Core/WinSWCore/WinSWCore.csproj

```diff
@@ -33,7 +33,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <SignAssembly>true</SignAssembly>
-    <AssemblyOriginatorKeyFile>$(SolutionDir)..\winsw_key.snk</AssemblyOriginatorKeyFile>
+    <AssemblyOriginatorKeyFile>winsw_key.snk</AssemblyOriginatorKeyFile>
```

Comment:
It does not work correctly on the solution level, at least in my IDE

Reply:
Done